daemon/execdriver/native/driver.go

```diff
@@ -335,7 +335,9 @@ func (d *driver) Clean(id string) error {
 }
 
 func (d *driver) Stats(id string) (*execdriver.ResourceStats, error) {
+	d.Lock()
 	c := d.activeContainers[id]
+	d.Unlock()
```

Comment:
I'm also trying to reproduce a race with pause/unpause (even though seems clear that accessing `activeContainer` from pause/unpause is racy)